rpc/handlers.go

```diff
@@ -633,3 +639,17 @@ func (h *Handler) ClassAt(id *BlockID, address *felt.Felt) (*Class, *jsonrpc.Err
 	}
 	return h.Class(id, classHash)
 }
+
+// AddInvokeTransaction relays an invoke transaction to the gateway.
+//
+// It follows the specification defined here:
+// https://github.com/starkware-libs/starknet-specs/blob/a789ccc3432c57777beceaa53a34a7ae2f25fda0/api/starknet_write_api.json#L11
+func (h *Handler) AddInvokeTransaction(broadcastInvokeTxn *gateway.BroadcastInvokeTxn) (*core.AddInvokeTxResult, *jsonrpc.Error) {
+	invokeResult, err := h.gatewayClient.AddInvokeTransaction(context.TODO(), broadcastInvokeTxn)
```

Comment:
Context with a timeout should be enough here IMO

Reply:
How long should the timeout be, 2 seconds?

Reply:
But +1 for timeout, just keep `context.TODO()` instead of `...Background()`